mysql-test/suite/rpl/t/rpl_binlog_sender_fseek.test

```diff
@@ -9,9 +9,9 @@
 # sometimes, cross the IO_CACHE block size. After running the workload, the test
 # case will verify the P_S counter COUNT_MISC for "wait/io/file/sql/binlog"
 # EVENT_NAME at performance_schema.file_summary_by_event_name. In a single
-# binary log file case, the counter should not be more than 5 (as there are some
+# binary log file case, the counter should not be more than 6 (as there are some
```

Comment:
Report the bug of PFS not registering seeks upstream

Reply:
Reported PS-3937 "Some file operations in mf_iocache2.c are not instrumented"
https://jira.percona.com/browse/PS-3937